src/iceberg/meson.build

```diff
@@ -196,6 +197,7 @@ install_headers(
         'transform.h',
         'type_fwd.h',
         'type.h',
+        'update/replace_sort_order.h',
```

Comment:
We need to add `iceberg/update/meson.build` to install update/*.h. Could you fix this?